Build.props

```diff
@@ -0,0 +1,29 @@
+<Project>
```

Comment:
https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1040?view=vs-2019 and https://rules.sonarsource.com/csharp/RSPEC-4023 are the same, please choose one to keep.

Reply:
I've disabled Sonar's S4023 rule. As for me the priority of these 3 analyzer packages is:

1. FxCop (as it is developed by Microsoft)
1. StyleCop (because it is being useful for many years)
1. Sonar

So I would better disable the duplicate rule of analyzer with lower priority.

Reply:
Agree, great.